assets/src/apps/authoring/components/PropertyEditor/schemas/screen.ts

```diff
@@ -148,19 +155,30 @@ export const transformScreenModeltoSchema = (activity?: IActivity) => {
           : '255, 255, 255'
       },${data.palette.fillAlpha || '100'})`,
     };
-    return {
+    let schemaData = {
       ...data,
       title: activity?.title || '',
       Size: { width: data.width, height: data.height },
       checkButton: { showCheckBtn: data.showCheckBtn, checkButtonLabel: data.checkButtonLabel },
       max: { maxAttempt: data.maxAttempt, maxScore: data.maxScore },
       palette: data.palette.useHtmlProps ? data.palette : schemaPalette,
     };
+    if (currentSequence?.custom.isBank) {
+      schemaData = {
+        ...schemaData,
+        bankShowCount: currentSequence.custom.bankShowCount,
```

Comment:
should probably add some defaults here
```suggestion
        bankShowCount: currentSequence.custom.bankShowCount || 1,
```